### PR TITLE
Document the API

### PR DESCRIPTION
**Pull request by bryanburgers**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,6 +60,291 @@ environment variables manually (the actual bus address might be different):
...
+Connect to a bus. `busName` must be either `"system"` to connect to the system
+bus or `"session"` to connect to the session bus.
+
+Returns a `Bus`.
+
+```
+var bus = dbus.getBus('session');
+```
+
+#### DBus.prototype.registerService(busName, serviceName)
```
Comment (Shouqun) [inline README.md]: also wrap it with backtick:  `DBus.prototype.registerService(busName, serviceName)`

Comment (bryanburgers) [inline README.md]: Ooops. Can't believe I missed that one.

Comment (bryanburgers) [inline README.md]: I just pushed up an amended commit with that change made.

Comment (Shouqun) [inline README.md]: Thanks!

**Pull request merged**